Makefile

```diff
@@ -17,38 +17,47 @@ GIT = git
 DOTNET = dotnet
 
 GITFLAGS = clone --depth=1
-DOTNETFLAGS = -v:q -nologo
+DOTNETFLAGS = -nologo -v:q -c:Debug
```

Comment:
i dont think we use debug on the windows install @valentinbreiz do you know?

Reply:
This is for Linux anyways

Reply:
yes but we should do the same as windows when building

Reply:
for build something with cosmos yes but if the windows build uses release to build cosmos we should do that on linux too

Reply:
We could make a setup for that, but for everything to be working for sure we will have to add this makefile to the github build action

Reply:
I think we use release mode on windows as well. We only use debug when compiling the kernel using cosmos